cmd/createorupdate/createorupdate.go

```diff
@@ -252,8 +252,7 @@ func main() {
 	adminClient.Authorizer = authorizer
 	v20180930previewClient.Authorizer = authorizer
 
-	ctx, cancel := context.WithTimeout(context.Background(), *timeout)
-	defer cancel()
+	ctx := fakerp.EnrichContext(context.Background())
```

Comment:
Remove the EnrichContext calls where they are not specifically needed (i.e. inside the fakepr process when it's going to pass that context into the plugin).  Remove this one and most if not all of the rest of the diff in this file will go back too.